site/src/docs/components/accordion/index.mdx

```diff
@@ -88,6 +90,23 @@ HDS Accordion includes three (3) size variants; small, default, and large. You c
   </Accordion>
 </PlaygroundPreview>
 
+#### Accordions, heading elements and `headingLevel`-property
+
+When using multiple Accordions as a group with a common header or the page content has a header preceeding the Accordion(s) keep in mind the basics of using the header elements, h1 being the most important header on the page down to h6 being the least important and give the Accordion the proper `headingLevel` according to it's position.
```

Comment:
That's a long sentence 😁 Here's Grammarly fixes (note the comma after "Accordion(s)"):
> When using multiple Accordions as a group with a **shared** header or the page content has a header **preceding** the Accordion(s)**,** keep in mind the basics of using the header elements, h1 being the most important header on the page down to h6 being the least important and give the Accordion the proper `headingLevel` according to **its** position.

Reply:
Yeah... fixiiing :D